catalog-rest-service/src/main/java/org/openmetadata/catalog/util/EntityUtil.java

```diff
@@ -172,6 +176,18 @@ public static EntityReference getService(
     return refs.isEmpty() ? null : refs.get(0);
   }
 
+  public static EntityReference getService(
+      EntityRelationshipDAO dao, String entityType, UUID entityId, String serviceType, Include include) {
+    List<EntityReference> refs =
+        dao.findFromEntity(
+            entityId.toString(), entityType, Relationship.CONTAINS.ordinal(), serviceType, toBoolean(include));
+    if (refs.size() > 1) {
```

Comment:
Did you mean > 0 or >= 1?

Reply:
> 1 is correct, right?

Reply:
I'm closing for now. Just reopen it if needed.